en/docs/troubleshoot/faq.md

```diff
@@ -0,0 +1,147 @@
+
+### Q: What is a trigger in Choreo Marketplace?
+A trigger is a construct that enables users to receive known event payloads from external systems, facilitating event-driven programming.
+
+### Q: What is a sample/template in Choreo?
+A sample or template is a prebuilt Ballerina program that covers a popular integration use case or pattern. Examples include connecting Salesforce to Slack or implementing content-based routing.
+
+### Q: What are the support options in Choreo?
+You can find information about our support plans, including `free`, `basic`, and `enterprise` options at [https://wso2.com/choreo/customer-support/](https://wso2.com/choreo/customer-support/).
+
+### Q: How can I perform log monitoring or analytics for the Azure environment?
+If you have a log monitoring product or service, such as Azure Monitor, you can use it together with Choreo. Note: The log monitoring tool is not included in the infrastructure cost.
+
+### Q: What source control software does Choreo support?
+Choreo currently supports GitHub. Support for Bitbucket and GitLab is on our roadmap. 
+
+### Q: What is Ballerina?
+Ballerina is an open-source programming language designed for the cloud. It simplifies the process of using, combining, and creating network services. When you use Ballerina to write integrations in Choreo, you can save time and deliver 2-3x faster. To learn more, check out https://ballerina.io/.
+
+### Q: What is Asgardeo?
+Asgardeo is an identity provider (IDP) that allows developers to secure access for consumers, business partners, employees, and APIs. Asgardeo is Choreo’s default IDP. To learn more, visit https://wso2.com/asgardeo/.
+
+## Security and Data Protection
+
+### Q: How is data managed in Choreo?
+Choreo manages data using WSO2 containers and Kubernetes clusters, which provide scalability, resilience, and security. Find out more [here](https://wso2.cachefly.net/wso2/sites/all/2023/pdf/wso2-public-cloud-data-protection-faq.pdf).
+
+### Q: What is the WSO2 Subprocessor list?
+This is a detailed list of all subprocessors used by WSO2, including their name, location, and purpose. This information is updated frequently to ensure compliance with data protection regulations and is found [here](https://wso2.cachefly.net/wso2/sites/all/2023/pdf/wso2-subprocessor-list.pdf).
+
+### Q: How do we secure WSO2 Private and Public Clouds?
+WSO2 uses a range of security controls and design patterns to protect against several threats, including internal attacks, software supply chain attacks, service and platform attacks, and more. Find out more regarding this [here](https://wso2.cachefly.net/wso2/sites/all/2023/pdf/securing-wso2-private-and-public-clouds.pdf).
+
+## Data Planes
+
+### Q: What is a Choreo Control Plane?
```

Comment:
Any reason to use title caps for the term control plane?

Reply:
No. Changed.